packages/react/src/components/table/table.tsx

```diff
@@ -72,6 +73,34 @@ function getHeading<T extends object>(column: HeaderGroup<T>, stickyHeader: bool
     );
 }
 
+const StyledFooter = styled.td<{ sticky: boolean }>`
+    ${({ sticky }) => sticky && css`
+        background-color: ${({ theme }) => theme.greys.white};
+        position: sticky;
+    `}
+`;
+
+function getFooter<T extends object>(column: HeaderGroup<T>, stickyFooter: boolean): ReactElement | null {
+    if (column.Footer === 'remove' || column.Footer === undefined || column.Footer === '') {
```

Comment:
If we don't want to render something, we should stick to the React way, which is `undefined` or `null`.

The problem with your code is the user cannot render `<td>remove</td>` or a blank cell `<td></td>`.

Reply:
We need that so we can have td with colspan, If I don't add possibility of removing it it will break.
Also if we don't use colspan cells and we have something long added to the cell it will stretch the entire column

Reply:
What I don't understand is we could just check undefined, why the 2 others conditions for?

Reply:
What's wrong with just `if (column.Footer === undefined)`? Why do we add the 2 other conditions? In case of colspan, the next cell(s) should be undefined, as in the html you don't create the TD as well.

Reply:
Why this was resolved, it's certainly a problem.

When `column.Footer: 'Total'`, it will output `<td>Total</td>`, but why `column.Footer: 'remove'` will not render `<td>remove</td>`? How do you render it then? Same with empty string, the user want a blank cell.

This is a magic string (https://en.wikipedia.org/wiki/Magic_string) and it's bad.

Reply:
ideally object without Footer would be displayed,
Footer with "remove" should be a way to remove the (<td></td>) and it will require the use of a colSpan.
if the user want's to keep the <td></td> because it wont affect col stretching they should keep it.
I need to remove the <td></td> every time the user needs to stretch the footer cell but not affect the entire column.
I'm going to remove the other conditions

Reply:
Done,
Now if the Footer prop is empt Footer: '';  the cell wont be render. this will be used with colSpan.

Reply:
Still not liking the '', so I ran your branch and now I understand better. Your concern is because when we use the footer data from the hooks, react-table had changed undefined column to `function emptyRenderer`.

I can think of 2 solutions to that

```
if (column.Footer.name === 'emptyRenderer') {
```

Or use the raw props.columns

Reply:
Even you "not linking" it, remember that sometimes you still want the td there even being empty because the content doesn't affect the width of the column.
I saw this possibility of using If column.Footer.name === 'emptyRenderer' as I read the docs, but I didn't use because all empty TDs will disappear forcing people to chase where to add the colSpan, instead of this being a logical decision when they create the footer.
hope it make sense.